projects/storefrontapp/src/app/app.module.ts

```diff
@@ -35,6 +35,7 @@ if (!environment.production) {
       backend: {
         occ: {
           baseUrl: environment.occBaseUrl,
+          prefix: environment.occPrefix,
```

Comment:
the occ prefix is pretty standard up to now and therefore we could avoid this additional env property

Reply:
This was not intended to be part of the PR, sorry for that. Was needed to test it with my local environment.
In a custom Commerce project, though, this is important to be set.